python/test/test_core.py

```diff
@@ -279,13 +280,14 @@ def test_extrinsic_free():
     # this test will pass if Queryable extends the lifetime of [] by holding a reference to it
 
 
-def test_user_distance():
+@pytest.mark.parametrize("new_distance,new_divergence", zip([dp.user_distance, _extrinsic_distance], [dp.user_divergence, _extrinsic_divergence]))
+def test_user_distance(new_distance, new_divergence):
```

Comment:
```suggestion
def test_distance(new_distance, new_divergence):
```

Reply:
`dp.user_distance` is repeated: Unless you intended for it to vary, but forgot, maybe move it inside the function and simplify the signature?